### DISCUSSION TITLE
Release date PU

### DISCUSSION THREAD

**Discussion by fue-medatixx** [Digitaler-Impfnachweis/certification-apis#42] [Q&A] [answered]
Is there a planed release date for PU ?

**Comment by oliver-steinbrecher** [comment, marked as answer]
PU connectivity should be in place end of this week.

**Reply by dirkexe** [reply]
Can you please confirm that the PU is now ready for production. We have a negative feedback here. Where can we search for the issue? At our PVS implementation or at IBM/ubirch?
